feeds/src/components/listing/GridItem.test.tsx

```diff
@@ -22,7 +22,7 @@ const AllTheProviders: React.FC = ({ children }) => {
 const feed = partialAsFull<FeedConfig>({
   name: 'pair name',
   path: '/link',
-  valuePrefix: 'prefix',
+  valuePrefix: '$',
```

Comment:
Please change `expect(container).toHaveTextContent('prefix')` to ` expect(container).toHaveTextContent('$')`